src/locales/en.json

```diff
@@ -50,6 +50,13 @@
         "synchronize": "Synchronize now"
       }
     },
+    "folder": {
+      "title": "Folder",
+      "link" : {
+        "default": "Open the %{name} folder",
```

Comment:
Sorry, I mean:
```json
"link" : {
  "default": "Open %{name} folder on Cozy Drive",
  [...]
```
By the way, why is there a different message when the konnector is fetching bills ?